cartesi-rollups/api/rollup/finish.api.mdx

```diff
@@ -25,7 +25,7 @@ When the Rollup HTTP Server returns 202, the DApp backend should retry the call
     report = {'payload': rollup_request['data']['payload']}\n        check_status_code(requests.post(rollup + '/report', json=report))\n\n        finish['status'] = 'accept'\n\n    elif rollup_request['request_type'] == 'inspect_state':\n        print('Sending report per inspect request')\n        report = {'payload': rollup_request['data']['payload']}\n        check_status_code(requests.post(rollup + '/report', json=report))\n\n    else:\n        print('Throwing rollup exception')\n        exception = {'payload': rollup_request['data']['payload']}\n        requests.post(rollup + '/exception', json=exception)\n        break\n```\n\nIn production mode, if the DApp exits the Rollups initialization script will register a Rollup Exception.\nSee [/exception](#api-Default-registerException).\n\nIn host mode, the Cartesi Rollups infrastructure is not able to detect that the DApp exited.\nIt is up to the DApp developer to re-launch the DApp.\n"},"postman":{"name":"Finish and get next request","description":{"content":"The DApp backend should call this method to start processing rollup requests.\nThe Rollup HTTP Server returns the next rollup request in the response body.\n\nThe possible values for the request_type field are 'advance_state' and 'inspect_state'.\nThe data field contains the rollup request input data.\nFor advance-state requests, the input data contains the advance-state metadata and the payload.\nFor inspect-state requests, the input data contains only the payload.\n\nAfter processing a rollup request, the DApp back-end should call again the finish method.\nFor advance-state requests, depending on the result of the request processing, it should fill the status field of the request body with 'accept' or 'reject'.\nThe Rollup HTTP Server ignores the content of the status field for the first finish request and after an inspect-state request.\n\nIf the advance-state request is rejected, the vouchers and notices are discarded.\nIn contrast, reports are not discarded in case of rejection.\nWhen running inside a Cartesi machine, the Cartesi Server Manager reverts the entire state of the machine to what it was before receiving the request.\n\nDuring a finish call, the next rollup request might not be immediately available.\nWhen the DApp backend and the Rollup HTTP Server are running inside a Cartesi machine, the Cartesi Server Manager pauses the whole machine execution until the next request is ready.\nWhen running in host mode, the Rollup HTTP Server returns the status code 202 after 10 seconds to avoid the connection timing out.\nWhen the Rollup HTTP Server returns 202, the DApp backend should retry the call to finish passing the same arguments as before.\n","type":"text/plain"},"url":{"path":["finish"],"host":["{{baseUrl}}"],"query":[],"variable":[]},"header":[{"key":"Content-Type","value":"application/json"},{"key":"Accept","value":"application/json"}],"method":"POST","body":{"mode":"raw","raw":"\"\"","options":{"raw":{"language":"json"}}}}}
```

Comment:
There was no change in this API. The diff is hard to read, but this seems wrong.